docs/java/features/extensions/extension-library/curconv/currency-conversion.mdx

```diff
@@ -46,6 +46,10 @@ Adding the Currency Conversion library to your list of dependencies will allow y
 
 Add the dependency in your pom.xml file under the existing *dependencies* tag as follows:
 
+:::note Dependency version is handled via Cloud SDK BOM
+When used as part of Cloud SDK for Java the version of extension dependencies is managed via BOM of Cloud SDK.
```

Comment:
```suggestion
When your project uses the SDK Bill-of-Material (BOM `sdk-modules-bom` in the dependencyManagement section, then you can omit the `version` tag. Otherwise, mention the latest SAP Cloud SDK version in the `version´ tag.
```